agent/agentscripts/agentscripts.go

```diff
@@ -314,6 +320,15 @@ func (r *Runner) run(ctx context.Context, script codersdk.WorkspaceAgentScript)
 		} else {
 			logger.Info(ctx, fmt.Sprintf("%s script completed", logPath), slog.F("execution_time", execTime), slog.F("exit_code", exitCode))
 		}
+
+		_, err = r.scriptCompleted(ctx, &proto.WorkspaceAgentScriptCompletedRequest{
```

Comment:
Are we only capturing timings here for startup scripts, or for all?

From https://github.com/coder/coder/issues/14630:

> Scripts which do not block (i.e. run on startup without start_blocks_login, on stop, or on a cron schedule) should also be tracked and have some indication that they are non-blocking.

Can you add in that detail please?